src/pages/resultsView/survival/SurvivalChart.spec.tsx

```diff
@@ -0,0 +1,184 @@
+import {assert} from 'chai';
+import {PatientSurvival} from "../../../shared/model/PatientSurvival";
+import {
+    getEstimates, getMedian, getLineData, getScatterData, getScatterDataWithOpacity, getStats, calculateLogRank,
+    getDownloadContent, convertScatterDataToDownloadData
+} from "./SurvivalChart";
+
+const exampleAlteredPatientSurvivals = [
+    {
+        patientId: "TCGA-OR-A5J1",
+        studyId: "acc_tcga",
+        months: 5.09,
+        status: false
+    },
+    {
+        patientId: "TCGA-OR-A5J2",
+        studyId: "acc_tcga",
+        months: 0.09,
+        status: true
+    },
+];
+
+const exampleUnalteredPatientSurvivals = [
+    
+    {
+        patientId: "TCGA-OR-A5J3",
+        studyId: "acc_tcga",
+        months: 0,
+        status: false
+    },
+    {
+        patientId: "TCGA-2F-A9KO",
+        studyId: "blca_tcga",
+        months: 63.83,
+        status: true
+    },
+    {
+        patientId: "TCGA-2F-A9KP",
+        studyId: "blca_tcga",
+        months: 0.13,
+        status: false
+    },
+    {
+        patientId: "TCGA-2F-A9KQ",
+        studyId: "blca_tcga",
+        months: 182.19,
+        status: true
+    }
+];
+
+const examplePatientSurvivals = exampleAlteredPatientSurvivals.concat(exampleUnalteredPatientSurvivals);
+
+const exampleAlteredEstimates = [1,0.8];
+const exampleUnalteredEstimates = [0.8,0.5333333333333333,0.5333333333333333,0];
+const exampleEstimates = exampleAlteredEstimates.concat(exampleUnalteredEstimates);
+
+describe("SurvivalChart", () => {
+    describe("#getEstimates()", () => {
```

Comment:
doesn't test the survivalStatus fork

Reply:
the example data has both status=true and status=false patients, so it enters both forks.